pdns/arguments.hh

```diff
@@ -106,9 +106,11 @@ public:
   void setCmd(const string &, const string &); //!< Add a command flag
   string &setSwitch(const string &, const string &); //!< Add a command flag
   string helpstring(string prefix=""); //!< generates the --help
-  string configstring(bool current=false); //!< generates the --mkconfig
+  string configstring(bool current=false, bool full=true); //!< generates the --mkconfig
```

Comment:
the pre-existing comment here is... outdated?

Reply:
looks like this PR fixes that now